sdk/schemaregistry/ci.yml

```diff
@@ -0,0 +1,29 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
```

Comment:
For my education, when/how is this generated? There has been manual edits to this since copied from template to get this library bootstrapped. Is there a better way to do that?

Reply:
I am not sure about this one.
@KarishmaGhiya ?

Reply:
The comment is a lie currently. We hand author these AFAIK.

Reply:
Should we have an issue to remove these comments, then?

Reply:
I filed #10434